### PR TITLE
added test naming conventions

### PR DESCRIPTION
**Pull request by mgretzke**
# Pull Request

## Description
Adding some test naming convention

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -127,7 +127,16 @@ Interfaces should be the entrypoint for all contracts. When exploring the a cont
 
 ## Testing
 
-The following testing practices should be followed when writing unit tests for new code. All functions, lines and branches should be tested to result in 100% testing coverage. Fuzz parameters and conditions whenever possible. Extremes should be tested in dedicated edge case and corner case tests. Invariants should be tested in dedicated invariant tests.
+The following testing practices should be followed when writing unit tests for new code:
```
Comment (gretzke) [inline CONTRIBUTING.md:130]: Test naming conventions

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -127,7 +127,16 @@ Interfaces should be the entrypoint for all contracts. When exploring the a cont
...
+- All functions, lines and branches should be tested to result in 100% testing coverage. 
+- Fuzz parameters and conditions whenever possible. 
+- Extremes should be tested in dedicated edge case and corner case tests. 
+- Invariants should be tested in dedicated invariant tests.
+
+Tests should follow clear naming principles to easily understand the purpose of the test:
+- The test itself (or the contract of the test) should include the name of the function it is testing.
+- When a test checks for a revert, the test name should start with `test_revert_` or `testRevert`.
```
Comment (zhongeric) [inline CONTRIBUTING.md:138]: what about when you want to test that it reverts with a specific error?

File: CONTRIBUTING.md (hunk 1 above)
Comment (gretzke) [reply CONTRIBUTING.md:130]: come back and expand the testing conventions
